### ISSUE TITLE
[A11Y] Download chart on the article landing page is missing link to an alternate data file

### ISSUE THREAD

**Issue by israelcefrin** [pkp/pkp-lib#9694]
## Issue Description
The article detailed page shows a chart with download stats of the submission (usually counting the PDF download). However, a screenreader user doesn't have access to the numbers since they are only embedded into the chart canvas.

## How to reproduce / Current behavior
1. Access a journal
2. Click on a published submission from the home page
3. Scroll down until "Downloads" section
4. It will show only the chart

## What application are you using?
 OJS version 3.4

## Stack used 
- Device: Laptop
- OS: MacOS
- Browser: Chrome

## Testing method
- Method: manual

## Action
It should be include a link to a downloadable format file, e.g. "Get the data"  -> CSV. It would enable any user that navigates the journal with a screen reader to access the data from the chart.

## Additional information

![Image](https://github.com/pkp/pkp-lib/assets/4992902/66d1a154-eb09-4f49-ae54-6062c113fd82)
